modules/nf-core/glimpse/ligate/main.nf

```diff
@@ -0,0 +1,36 @@
+process GLIMPSE_LIGATE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::glimpse-bio=1.1.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/glimpse-bio:1.1.1--hce55b13_1':
+        'quay.io/biocontainers/glimpse-bio:1.1.1--hce55b13_1' }"
+
+    input:
+    tuple val(meta), path(input)
+
+    output:
+    tuple val(meta), path("*.{vcf,bcf}"), emit: merged_variant
```

Comment:
nitpicking, but is there a compressed option for vcf?

Reply:
That's a good question, you can compressed it with bgzip.
I didn't tried yet if it was possible directly in the software.

Reply:
So apparently it's possible you just need to specify the extension.